src/backend/renderer/mod.rs

```diff
@@ -198,7 +214,9 @@ pub trait Renderer {
     /// Texture Handle type used by this renderer.
     type TextureId: Texture;
     /// Type representing a currently in-progress frame during the [`Renderer::render`]-call
-    type Frame: Frame<Error = Self::Error, TextureId = Self::TextureId>;
+    type Frame<'a>: Frame<Error = Self::Error, TextureId = Self::TextureId> + 'a
```

Comment:
do we want to use a more descriptive lifetime name than `'a`?

Reply:
Used `'frame`